pyproject.toml

```
@@ -1,5 +1,8 @@
 [project]
 name = "qem-bot"
+version = "42"
+description = "Tool for scheduling maintenance tests on openQA based on [SMELT](https://tools.io.suse.de/smelt) incidents and Gitea PRs."
+readme = "Readme.md"
```

Can we make CI fail when fields are missing?